src/components/relationships/AsyncInvitationsList.tsx

```
@@ -0,0 +1,297 @@
+                            >
+                              {isRevoking ? 'Revoking...' : 'Revoke Invitation'}
+                            </DropdownItem>
+                          </DropdownMenu>
+                        </Dropdown>
+                      )}
+
+                      {invitation.status === 'accepted' && (
+                        <Tooltip content="This user has joined">
+                          <span className="text-success">
+                            <svg className="h-5 w-5" fill="currentColor" viewBox="0 0 20 20">
+                              <path
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)